modules/eco-node-health-check-operator-creating-node-health-check.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-health-check-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-node-health-check-operator-creating-node-health-check_{context}"]
+= Creating a node health check
+Using the web console, you can create a node health check to identify unhealthy nodes and specify the remediation type and strategy to fix them.
+
+.Procedure
+
+. From the *Administrator* perspective of the {product-title} web console, click *Compute* -> *NodeHealthChecks* -> *CreateNodeHealthCheck*.
+. Specify whether to configure the node health check using the *Form view* (the default setting) or the *YAML view*.
```

Comment:
It's better not to say that Form view is the default setting. NHC plugin remembers last used 'view' and next time will start from this

Reply:
> It's better not to say that Form view is the default setting. NHC plugin remembers last used 'view' and next time will start from this

Thanks @anna-savina. I have removed the reference to the default setting.